instrumentation/all/opentelemetry-instrumentation-all.gemspec

```diff
@@ -42,15 +42,15 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'opentelemetry-instrumentation-koala', '~> 0.19.0'
   spec.add_dependency 'opentelemetry-instrumentation-lmdb', '~> 0.21.0'
   spec.add_dependency 'opentelemetry-instrumentation-mongo', '~> 0.21.0'
-  spec.add_dependency 'opentelemetry-instrumentation-mysql2', '~> 0.21.0'
+  spec.add_dependency 'opentelemetry-instrumentation-mysql2', '~> 0.21.1'
   spec.add_dependency 'opentelemetry-instrumentation-net_http', '~> 0.21.0'
-  spec.add_dependency 'opentelemetry-instrumentation-pg', '~> 0.22.0'
+  spec.add_dependency 'opentelemetry-instrumentation-pg', '~> 0.22.1'
   spec.add_dependency 'opentelemetry-instrumentation-que', '~> 0.4.0'
```

Comment:
```suggestion
  spec.add_dependency 'opentelemetry-instrumentation-que', '~> 0.5.0'
```

Reply:
Thanks! Once the release actually finishes we'll click this commit button. 😄